awscli/customizations/cloudformation/yamlhelper.py

```diff
@@ -12,6 +12,7 @@
 # language governing permissions and limitations under the License.
 
 import yaml
+import six
```

Comment:
So for ``six`` could you import from ``awscli.compat`` instead?: https://github.com/aws/aws-cli/blob/develop/awscli/compat.py#L17 The CLI does not have a direct dependency on ``six`` which would be the case for this import, but instead relies on botocore's vendored version of ``six`` and imports that in the ``compat`` module